src/core/doc_generators/models.py

```
@@ -592,22 +581,22 @@ def process_batch(self, config: BatchGenerationConfig) -> BatchGenerationResult:
+        output_path: str | Path,
+        _config: DocumentConfig | None = None,  # 将来の拡張機能用に予約済みパラメータ
     ) -> BatchGenerationResult:
         """
         ディレクトリ内のファイルを一括処理します。
 
         Args:
             input_directory: 入力ディレクトリ
-            output_directory: 出力ディレクトリ
-            config: ドキュメント設定（Noneの場合、デフォルト設定を使用）
+            output_path: 出力ディレクトリ
+            _config: ドキュメント設定（将来の拡張機能用に予約済み、現在未使用）
```

Ruff (0.14.0)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)